src/ontology/uberon-edit.obo

```diff
@@ -224180,6 +224180,16 @@ is_a: UBERON:0000029 ! lymph node
 relationship: dc-contributor http://orcid.org/0000-0002-1773-2692
 property_value: dcterms-date 2022-05-26T11:59:51Z xsd:dateTime
 
+[Term]
+id: UBERON:8480000
+name: iliac vein smooth muscle tissue
+def: "A portion of smooth muscle tissue that is part of an iliac vein." []
+synonym: "muscle of iliac vein" BROAD []
```

Comment:
synonyms should ideally come with reference too (I know @bvarner-ebi requested dbxref for def already, but synonyms should ideally have them too)